src/styles/spacing.ts

```diff
@@ -10,12 +10,17 @@ export const spacing = {
 	20: '1.25rem',
```

Comment:
we should rethink if this might be too many keys by now ... if we add more we don't need the spacing anymore :)